### PR TITLE
[Spritelab] Enable Start in Animation Tab level configuration

### PR DESCRIPTION
**Pull request by ajpal**
First change: Don't dispatch the `changeInterfaceMode` event until after we've called `StudioApp.init`, which ensures that Blockly has been initialized properly. This fixes the following JS errors:
image

Second change: Rerender Blockly.mainBlockSpace when switching from the costume tab to the code tab. This render only seems necessary the first time you switch from costume to code, but we don't know whether it's necessary from within the change handler, so I have it just always re-rendering. I don't think that'll hurt performance because it's just one render, and Blockly rerenders things basically any time you drag a block anyways.

Before:
initial page load:
image
switch to code:
image
Then if you resize the window or the instructions/playspace panes:
image

After:
initial page load:
image
switch to code:
image

initial page load:
image
switch to code:
image

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-1402)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/P5Lab.js [hunk 1]
```diff
@@ -359,6 +359,10 @@ P5Lab.prototype.init = function(config) {
 
     this.studioApp_.init(config);
 
+    if (startInAnimationTab(getStore().getState())) {
```
Comment (maddiedierker) [inline apps/src/p5lab/P5Lab.js:362]: i'm not sure i understand how moving this conditional from line 449 to line 362 fixed the issue we were seeing since `this.studioApp_.init(config)` is on line 360

Comment (ajpal) [reply apps/src/p5lab/P5Lab.js:362]: it's a little hard to follow because P5Lab.init() has *a lot* going on, but `this.studioApp_.init(config)` is inside another function (`onMount`), which is defined on line 324. So I'm moving the conditional to be after `this.studioApp_.init(config)` in that function, which is passed to P5LabView and called after that component is mounted

**Review by maddiedierker — APPROVED** [review]
